### PR TITLE
I can read documentation about setting up a cluster

### LINKED ISSUE #1453: I can read documentation about setting up a cluster

### PR DESCRIPTION
**Pull request by ChrisKujawa**
I'm not 100% sure if this enough, but currently I have no idea what I could add.

closes #1453

### CODE REVIEW

File: docs/src/operations/setting-up-a-cluster.md [hunk 1]
```diff
@@ -1,3 +1,172 @@
...
+# initialContactPoints = []
+```
+
+# Example
+
+In this example we will setup an Zeebe cluster with
+five brokers. Each broker needs to get an unique node id.
+To scale well, we will bootstrap five partitions
+with an replication factor of three. For more information about this,
+please take a look into the [Clustering](/basics/clustering.md) section.
```
Comment (menski) [inline docs/src/operations/setting-up-a-cluster.md]: This link is dead for me locally, where should it point to?

File: docs/src/operations/setting-up-a-cluster.md [hunk 2]
```diff
@@ -1,3 +1,172 @@
...
+nodeId = 0
+partitionsCount = 5
+replicationFactor = 3
+clusterSize = 5
+```
+
+For the other brokers the configuration will slightly change.
+```toml
+[cluster]
+nodeId = 1
```
Comment (menski) [inline docs/src/operations/setting-up-a-cluster.md]: Maybe this should also be a placeholder as the user has to change it.

**Review by menski — CHANGES_REQUESTED** [review]

File: docs/src/operations/setting-up-a-cluster.md (hunk 1 above)
Comment (Zelldon) [reply docs/src/operations/setting-up-a-cluster.md]: http://localhost:3000/basics/clustering.html

File: docs/src/operations/setting-up-a-cluster.md (hunk 2 above)
Comment (Zelldon) [reply docs/src/operations/setting-up-a-cluster.md]: Hm yeah can do. I also explained that is has to be incremented and between 0 and clustersize -1

**Comment by Zelldon** [comment]
@menski Please take a look into the latest changes.

**Review by menski — APPROVED** [review]

**Comment by menski** [comment]
@Zelldon thanks. Feel free to merge

**Pull request merged**